### PR TITLE
Fix event ordering.  

### PR DESCRIPTION
**Pull request by briandorsey**
Implied sort based on filesystem was inconsistent, moved to a single yaml file.

### CODE REVIEW

**Comment by roberthbailey** [comment]
Thanks Brian! Do you know if this will automatically update the site once it's merged?

**Comment by briandorsey** [comment]
Yes, all commits to gh-pages kick-off a hosted Jekyll run to make the static site.

**Comment by briandorsey** [comment]
Staged version of the changes on my account here: http://briandorsey.github.io/kubernetes/events/

File: _layouts/events.html [hunk 1]
```diff
@@ -39,7 +37,7 @@ <h3 class="alt text-gray">{{event.title}}</h3>
         {{event.venue}}
        </div>
        <div class="speaker">
-        {{event.speaker}}
+        {{event.speaker}}  is speaking
```
Comment (roberthbailey) [inline _layouts/events.html]: The english here is incorrect if event.speaker has more than one name in it (e.g. "Tim & Dawn is speaking")

Comment (roberthbailey) [inline _layouts/events.html]: To be clear, I'd like to see this either left the way it was or properly switch between 'is' and 'are' depending on the number of speakers.

**Comment by roberthbailey** [comment]
LGTM. Can you squash your commits before I merge?

**Comment by briandorsey** [comment]
Squashed.

**Pull request merged**